ui/hooks/organismes.ts

```diff
@@ -103,3 +104,65 @@ export function useOrganismesFiltered(organismes: OrganismeNormalized[]) {
     organismesFiltered,
   };
 }
+
+export function useOrganismesNormalizedLists(organismes: Organisme[]) {
+  const { organismesFiables, organismesACompleter, organismesNonRetenus, nbOrganismesFermes } = useMemo(() => {
+    const organismesFiables: OrganismeNormalized[] = [];
+    const organismesACompleter: OrganismeNormalized[] = [];
+    const organismesNonRetenus: OrganismeNormalized[] = [];
+
+    let nbOrganismesFermes = 0;
+    (organismes || []).forEach((organisme: OrganismeNormalized) => {
+      // We need to memorize organismes with normalized names to be avoid running the normalization on each keystroke.
+      organisme.normalizedName = normalize(organisme.enseigne ?? organisme.raison_sociale ?? "");
+      organisme.normalizedUai = normalize(organisme.uai ?? "");
+      organisme.normalizedCommune = normalize(organisme.adresse?.commune ?? "");
+
+      if (organisme.fiabilisation_statut === "FIABLE" && !organisme.ferme && organisme.nature !== "inconnue") {
+        organismesFiables.push(organisme);
+      } else if (
+        // Organismes à masquer :
+        // organismes fermés et ne transmettant pas
+        // organismes inconnus (sans raison sociale ni enseigne) et absents du référentiel ou fermé
+        (organisme.ferme && !organisme.last_transmission_date) ||
+        (!organisme.enseigne &&
+          !organisme.raison_sociale &&
+          (organisme.est_dans_le_referentiel === "absent" || organisme.ferme))
+      ) {
+        nbOrganismesFermes++;
+        organismesNonRetenus.push(organisme);
+      } else {
+        organismesACompleter.push(organisme);
+        if (organisme.ferme) {
+          nbOrganismesFermes++;
+        }
+      }
+    });
+
+    return {
+      organismesFiables,
+      organismesACompleter,
+      organismesNonRetenus,
+      nbOrganismesFermes,
+    };
+  }, [organismes]);
+
+  return {
+    organismesFiables,
+    organismesACompleter,
+    organismesNonRetenus,
+    nbOrganismesFermes,
+  };
+}
+
+export function useOrganismesDuplicatsLists() {
+  const router = useRouter();
+
+  const { data: organismesDuplicats, isLoading } = useQuery<Organisme[], any>(
+    ["admin/organismes-duplicats"],
```

Comment:
Synchronisation avec la route :)
```suggestion
    ["admin/organismes-duplicates"],
```